server/src/test/java/org/elasticsearch/indices/recovery/RecoverySourceHandlerTests.java

```diff
@@ -438,7 +438,6 @@ protected void failEngine(IOException cause) {
             handler.sendFiles(store, metas.toArray(new StoreFileMetaData[0]), () -> 0);
             fail("exception index");
         } catch (RuntimeException ex) {
-            assertNull(ExceptionsHelper.unwrapCorruption(ex));
```

Comment:
Just removed this as the underlying corruption exception is now unwrapped here propery (it's suppressed on the `RuntimeException`)